src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -50,6 +52,77 @@ def __init__(self, quantization_config: CompressedTensorsConfig, **kwargs):
         self.run_compressed = quantization_config.run_compressed
         self.quantization_config = quantization_config
 
+    def update_missing_keys_after_loading(self, model, missing_keys: List[str], prefix: str) -> List[str]:
+        """
+        Update missing keys after loading the model. This is necessary for compressed tensors
+        to load the model correctly. We expect weights to be present in missing keys.
+        The weight's are re-constructed by ModelCompressor in _process_model_after_weight_loading
+
+        This function cleans up expected missing keys and returns the remaining missing keys
+        """
+
+        from compressed_tensors import CompressionFormat
+
+        sparse_compressed = (
```

Comment:
I generally think we should have the logic of unexpected/missing keys live in ct, as opposed to having to maintain the list here

Reply:
I really like this idea; thank you!

Reply:
Updated!